src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml

```diff
@@ -215,7 +84,172 @@
     </Button.Template>
 
     <Button.ContextMenu>
-        <!-- Ensure the context menu is not null -->
-        <ContextMenu Style="{StaticResource ContextMenuStyle}"/>
+        <ContextMenu ItemsSource="{Binding MenuItemViewModels}"
```

Comment:
Were these just moved or also changed? Hard to diff with the large amount of movement in the file.

Reply:
Most of the changes are just moving (From the button.resource => contextMenu.resouces). 
I can comment the real changes in the xaml file.